book/website/community-handbook/github-gardening.md

```diff
@@ -0,0 +1,197 @@
+(ch-gardening)=
+# GitHub gardening
+
+Managing a open source community and maintaining its infrastructure, like [code repositories](#rr-vcs) and {term}`issue trackers <Issue Tracking>`, takes time an effort.
+In the free and open source community this work is often called maintenance.
+This work is difficult and requires strong technical skills as well as excellent interpersonal skills.
+It can often seem that this work goes unappreciated, but it is critically important for the health of an open source project.
+
+In _The Turing Way_ we call this work gardening, because that is an analogy for how this kind of activity is about tending to the community, keeping it healthy and helping it grow.
+This page has [instructions](#ch-gardening-checklists) and guidelines for our gardeners to help them work.
+
+## Guiding principles
+
+### Maintain quality
+
+Gardening work plays an important role in maintaining the quality of a project.
+[Continuous integration processes](#rr-ci) and [code review](#rr-reviewing) help to ensure the quality of contributions as they are merged.
+However, there are many other aspects of the community where gardeners can help.
+
+You should focus on keeping the quality of issues, discussions and pull requests high.
```

Comment:
I think I would continue talking here and below about the gardeners ("Gardeners should  focus ...") to flow with what has been said previously.